packages/sil-governance/src/routes/show-electorate/components/ElectorateMembers.tsx

```diff
@@ -1,37 +1,37 @@
 import { Electorate } from "@iov/bns";
+import { Table, TableBody, TableCell, TableHead, TableRow } from "@material-ui/core";
 import { Block, Typography } from "medulas-react-components";
-import React from "react";
+import React, { useMemo } from "react";
 
 interface Props {
   readonly electorate: Electorate;
 }
 
 const ElectorateMembers = ({ electorate }: Props): JSX.Element => {
-  const MembersTable = (): JSX.Element => (
-    <table>
-      <thead>
-        <tr>
-          <th>
-            <Typography>Address</Typography>
-          </th>
-          <th>
-            <Typography>Weight</Typography>
-          </th>
-        </tr>
-      </thead>
-      <tbody>
-        {Object.keys(electorate.electors).map(address => (
-          <tr key={address}>
-            <td>
-              <Typography>{address}</Typography>
-            </td>
-            <td>
-              <Typography>{electorate.electors[address].weight}</Typography>
-            </td>
-          </tr>
-        ))}
-      </tbody>
-    </table>
+  const MembersTable = useMemo(
```

Comment:
Move this into separate component please. You need to use `useMemo` for different complex calculations and do not use it as component substitution.